spiketools/stats/shuffle.py

```diff
@@ -75,15 +75,15 @@ def shuffle_isis(spikes, n_shuffles=1000):
     return shuffled_spikes
 
 
-def shuffle_bins(spikes, bin_width_range=(50, 2000), n_shuffles=1000):
+def shuffle_bins(spikes, bin_width_range=[.5, 7], n_shuffles=1000):
     """Shuffle data with a circular shuffle of the spike train.
```

Comment:
This description (from before) seems to be not quite right - I think it was copied from shuffle_circular, but it should really say something about being a binned shuffling approach.